src/lib/containers/table/table-top/DownloadOptions.tsx

```diff
@@ -67,12 +67,17 @@ export const DownloadOptions: React.FunctionComponent<
           </Dropdown.Item>
           {isFileViewOrDataset && (
             <Tooltip
-              title="A draft version of a dataset cannot be added to the Download Cart"
+              title={
+                disableDownload
+                  ? 'A draft version of a dataset cannot be added to the Download Cart'
+                  : null
+              }
               placement="left"
               enterNextDelay={300}
-              disableHoverListener={!disableDownload}
+              describeChild={true}
```

Comment:
I think in v5 they changed it so the child's `aria-label` attribute is set to the Tooltip's `title` prop, unless you set `describeChild` to be true (which is appropriate in this case, because it is _describing_ the child, not _labeling_ it). This change fixed broken tests.